boards/PDM/Src/Io/Io_CurrentSense.c

```diff
@@ -1,10 +1,13 @@
 #include "Io_CurrentSense.h"
-
-// TODO: Fix hard-coded values once the ADC is configured to read load-switches
+#include "Io_Adc.h"
+#include "main.h"
 
 float Io_CurrentSense_GetAux1Current(void)
 {
-    return 0.5f;
+    // Aux1 Current = ADC Voltage * Current Gain Ratio
+
+    const float LOW_CURRENT_SENSE_GAIN_RATIO = 500.0f;
+    return (Io_Adc_GetChannel6Voltage() * LOW_CURRENT_SENSE_GAIN_RATIO);
```

Comment:
You should be able to include the calculation for Aux2's current for this PR as well right ?